ORST/Assets/Scripts/Core/Movement/AdvancedTeleportTargetHandlerNode.cs

```diff
@@ -0,0 +1,104 @@
+using UnityEngine;
+using System.Collections;
+
+namespace ORST.Core {
+    public class AdvancedTeleportTargetHandlerNode : TeleportTargetHandlerNode {
+
+        [SerializeField] private GameObject invalidTeleportPosIndicator;
+        [SerializeField] private bool showInvalidPositionIndicator = true;
```

Comment:
These two don't match the naming convention for private fields
```suggestion
        [SerializeField, Required, ShowIf(nameof(m_ShowInvalidPositionIndicator))] private GameObject m_InvalidTeleportPosIndicator;
        [SerializeField] private bool m_ShowInvalidPositionIndicator = true;
```